crates/api_decl/src/namespaces/anvil_ext.rs

```diff
@@ -0,0 +1,11 @@
+//! This module extends the Anvil JSON-RPC API with anvil-zksync specific RPC methods.
+
+use jsonrpsee::core::RpcResult;
+use jsonrpsee::proc_macros::rpc;
+use zksync_types::{L1BatchNumber, H256};
+
+#[rpc(server, namespace = "anvil")]
```

Comment:
Discussion point: what is appropriate namespace?
- `anvil` -- pro is that it's a "natural" scope, con is that it's an extension that will not be covered in `anvil` docs.
- `anvil_zksync` -- pro is that it's super explicit, con is that we start having too many namespaces
- `zks` -- pro is that it's zksync-native namespace, con is that it's not available on the main node.

Subjectively I'm leaning towards `zks`.

Reply:
One counter-argument of extending `zks`: presumably we won't stop at `commit`/`prove`/`execute` (which are all clearly testing-related) so we might create false expectations of some of the future methods being available on ENs.

Either of the first two options seem fine to me. Slightly leaning toward a new namespace as I hated what early ETH tooling has done to the `evm` namespace lol. We could shorten `anvil_zksync` to `anvil_zks` for brevity if that helps

Reply:
`anvil_zks` ftw